### PR TITLE
Patch sql param generation

### PR DESCRIPTION
**Pull request by rgwozdz**
When `returnGeometry=false`, the generated SQL doesn't include call to reprojection function; As a result we should not add CRS arguments to the array of sql parameters.

### CODE REVIEW

File: lib/sql-query-builder/create-sql-string.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
```
[the remark this answers is not in the document]
Comment (rgwozdz) [reply lib/sql-query-builder/create-sql-string.js]: Fixed.

File: lib/sql-query-builder/create-sql-params.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  return params
+}
+
+function shouldAddCrsParams (options = {}) {
+  const {
+    aggregates,
+    returnGeometry,
+    inputCrs,
+    outputCrs
+  } = options
```
[the remark this answers is not in the document]
Comment (rgwozdz) [reply lib/sql-query-builder/create-sql-params.js:38]: done

File: lib/sql-query-builder/create-sql-string.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (robertvo824) [reply lib/sql-query-builder/create-sql-string.js]: Did changes from the last commit apply? I don't see any changes for some reason when I look at the files.

[the remark this answers is not in the document]
Comment (rgwozdz) [reply lib/sql-query-builder/create-sql-string.js]: Try now, there was a temporary problem.

**Review by robertvo824 — APPROVED** [review]

**Pull request merged**